submissions/sofiichuk/pop-up-task/style.css

```diff
@@ -0,0 +1,119 @@
+* {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    list-style: none;
+}
+
+.navBar>ul {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+    justify-content: end;
+    margin: 20px 40px 0 0;
+ }
+
+a, span {
+    text-decoration: none;
+    color: #4f4f4f;
+    width: 60px;
+    font-family: sans-serif;
+}
+
+.dropdownMenu a, span {
+    display: block;
+}
+
+img {
+    height: 35px;
+    width: auto;
+}
+
+img[src="icons/bell.png"] {
+    height: 28px !important;
+    width: 28px !important;
```

Comment:
Avoid using `!important` in your styles.

Reply:
Removed that.